lxd/storage/backend_lxd_patches.go

```diff
@@ -155,3 +156,63 @@ DELETE FROM storage_volumes WHERE id IN (
 
 	return nil
 }
+
+// patchBucketNames modifies the naming convention of bucket volumes by adding
+// the corresponding project name as a prefix.
+func patchBucketNames(b *lxdBackend) error {
+	// Apply patch only for btrfs, dir, lvm, and zfs drivers.
+	driverName := b.driver.Name()
+	if driverName != "zfs" && driverName != "btrfs" && driverName != "dir" && driverName != "lvm" {
```

Comment:
we have shared.StringInSlice which would be clearer here I think

Reply:
It will be clearer. Thanks